plinth/__main__.py

```diff
@@ -82,6 +83,14 @@ def setup_logging():
     cherrypy.log.error_file = cfg.status_log_file
     cherrypy.log.access_file = cfg.access_log_file
 
+    # Capture all Python warnings such as deprecation warnings
+    logging.captureWarnings(True)
+
+    # Log all deprecation warnings when in debug mode
+    if cfg.debug:
+        warnings.filterwarnings('default', '', DeprecationWarning)
+        warnings.filterwarnings('default', '', PendingDeprecationWarning)
+
```

Comment:
One last question from me: https://docs.python.org/3/library/warnings.html lists also ImportWarnings are ignored by default; maybe they should also be enabled?